app/queries/crash.ts

```diff
@@ -224,14 +224,12 @@ export const GET_CRASH = gql`
       }
       crash_notes(
         where: { is_deleted: { _eq: false } }
-        order_by: { date: desc }
+        order_by: { updated_at: asc }
```

Comment:
would you sort these by `created_at` instead?

Reply:
ah yes, that's terrible. let's show `created_at` in the header instead. that way the notes are sorted correctly, and when you edit one it's sort position doesn't jump. sorry for the back and forth!